include/blmc_drivers/blmc_joint_module.hpp

```diff
@@ -231,6 +231,16 @@ class BlmcJointModule
                    double& index_angle,
                    bool mechanical_calibration = false);
 
+    /**
+     * @brief Set zero position relative to current position
+     *
+     * @param joint_id ID of the joint.  This is only used for debug prints.
+     * @param home_offset_rad  Offset from home position to zero position.
+     *     Unit: radian.
+     */
```

Comment:
Btw: I like how well commented and formatted your code is! :)

Reply:
Thanks :)
I also wrote an interface for Solo8. Would you be interested in that as well?

Reply:
I will probably not use it (at least in the near future), but it might still be nice to add it to robot_fingers, so everything is in one place.  I think that would make it easier to maintain it.